src/data/constants/instructorInfo.js

```diff
@@ -0,0 +1,8 @@
+export const FAIL_INSTRUCTOR_INFO = 'FAIL_INSTRUCTOR_INFO';
+export const RECEIVE_INSTRUCTOR_INFO = 'RECEIVE_INSTRUCTOR_INFO';
+export const REQUEST_INSTRUCTOR_INFO = 'REQUEST_INSTRUCTOR_INFO';
```

Comment:
For just the initial create page, you won't need to bother with requesting info. Those three actions can be dropped for this initial PR (though they'll come back later).

Reply:
Started pulling this thread, but it kinda snowballs and we're just going to need to add it all back again in for DISCO-682 (or one of those stories)